addon/components/item-picker/template.hbs

```diff
@@ -40,7 +40,7 @@
         {{/if}}
       </ul>
 
-      {{#if currentModel}}
+      {{#if currentModel.item}}
```

Comment:
Previously we had `{{#if currentModel}}` which implies that `currentModel` could be `null` or `undefined`. If that is true (I don't know that it is), this change will break.

Reply:
I don't think it should ever be.

`currentModel` is always an `{}`, which is why this was originally watching the `item`. Because if the `item` was null, it wouldn't show anything. If there was something in it, it would be active.

Reply:
I think that's part of the reason why the side panel wouldn't close. Because nothing was ever changing (it always saw `{}` instead of `null` vs an item)